.docker/parameters.yml

```diff
@@ -31,17 +31,15 @@ parameters:
     xpub_client_id: some-id
     xpub_client_secret: ThisIsASecret
     subject_rewrites: []
-    example_rds_articles:
-        '16846': https://repro.elifesciences.org/example.html
     rds_articles:
```

Comment:
We can't get rid of this reference yet because this is set in `journal-formula` and `builder-configuration`.